java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -1635,11 +1381,8 @@ public ParenthesizedTreeImpl parenthesizedExpression(InternalSyntaxToken leftPar
   }
 
   public ExpressionTree newExpression(JavaTree newToken, Optional<List<AnnotationTreeImpl>> annotations, ExpressionTree partial) {
-    if (annotations.isPresent()) {
-      ((JavaTree) partial).prependChildren(annotations.get());
-    }
+    // FIXME SONARJAVA-547 Handle annotations
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)